ansible/roles/telegraf/tasks/main.yml

```diff
@@ -0,0 +1,80 @@
+---
+# Copyright (C) 2021 Dr. Serge Victor <https://dr.sergevictor.eu/>
+# Copyright (C) 2020 DebOps <https://debops.org/>
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- name: Import custom Ansible plugins
+  import_role:
+    name: 'ansible_plugins'
+
+- name: Import DebOps global handlers
+  import_role:
+    name: 'global_handlers'
+
+- name: Divert the original telegraf main configuration file
+  dpkg_divert:
+    path: '/etc/telegraf/telegraf.conf'
+    state: 'present'
+    delete: true
```

Comment:
In DebOps we use capitalized `True`/`False` for booleans, just like it's done in Python.

Since there is no mechanism to uninstall Telegraf implemented, the role will never remove the diversion. The `delete` parameter is not needed here.